test/unit/actions/save.test.js

```diff
@@ -1,9 +1,10 @@
 /* @flow */
 
 import { ZalgoPromise } from "@krakenjs/zalgo-promise/dist/zalgo-promise";
-import { describe, it, expect } from "vitest";
+import { describe, it, expect, vi } from "vitest";
 
 import { createSaveAction } from "../../../src/actions/save"
+import type { onErrorCallback } from "../../../src/actions/save"
```

Comment:
Linting breaks on having this import done twice, but Flow breaks if I try to combine them. I'm not sure the solution. We _could_ have an explicit `types.js` file wherein we define the types and use them in both places. Thoughts? I'm sure there is some Flow-fu I'm missing here, too.